crates/next-core/src/next_font_google/mod.rs

```diff
@@ -212,6 +216,23 @@ impl ImportMappingReplacement for NextFontGoogleCssModuleReplacer {
     }
 }
 
+async fn get_request_id(query_vc: QueryMapVc) -> Result<StringVc> {
+    let query = &*query_vc.await?;
+    let query = query.as_ref().context("Query map must be present")?;
+    let mut to_hash = vec![];
```

Comment:
Nit: There's no need to build a vec for this, using a `Xxh3Hash64Hasher` instance will allow you to compute this piecemeal.

Also, is this query sorted? We should consider `?a=b&c=d` to be equal to `?c=d&a=b` for hashing.